antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/mode/common/binding_const.py

```diff
@@ -8,21 +8,13 @@
 
 class OutputSimulationBindingConstraintItem(FolderNode):
     def build(self) -> TREE:
-        # filters = self.config.get_filters_synthesis(self.area, self.link)
-        # todo get the config related to this output (now this may fail if input has changed since the launch)
-
-        freq: MatrixFrequency
+        existing_files = [d.stem.replace("binding-constraints-", "") for d in self.config.path.iterdir()]
```

Comment:
Cette partie de code n'est pas couverte par les tests unitaires.

Reply:
I'll try to add some